### PR TITLE
added user promt to ask if a data set should be downloaded again  if it already exists locally

### PR DESCRIPTION
**Pull request by markuswnuk91**
Found this adaptation helpful, since for large data sets, the script downloads all repositories which are configured in darus_config.json. While I like that one is able to store all data sets relevant to a project in the darus_config.json, the run time of the script becomes quickly very long for large data sets (~1GB). Being promted if data sets shoud be redownloaded if it already exists locally makes it easier to update the project with a new data set, while making it not necessary to redownload all data that might already exist.

However, this might be a specific adaptiation useful for my workflow and not applicable to other workflows.
So maybe we should consider other workflows before merging.

### CODE REVIEW

File: scripts/darus_config.json
Comment (chhinze) [inline scripts/darus_config.json:1]: This file should not be checked in, right?

Comment (chhinze) [reply scripts/darus_config.json:1]: Looks good to me otherwise.

**Comment by chhinze** [comment]
Looks good to me otherwise

File: scripts/darus_config.json
Comment (markuswnuk91) [reply scripts/darus_config.json:1]: Ups, sorry. 
Checked in the config file to keep track of reposiories in my project. You are right here it should not be checked in

**Comment by chhinze** [comment]
Awesome. Thanks for your contribution.

**Pull request merged**